src/rdo.rs

```diff
@@ -422,6 +425,8 @@ fn compute_tx_distortion<T: Pixel>(
     (tx_dist as f64 * bias) as u64
   };
 
+  distortion = (fi.dist_scale[0] * distortion as f64) as u64;
```

Comment:
Thus, if not skipped block, this seems wrong to me since distortion is already has been scaled.